proto/persistenceblobs/server_message.proto

```diff
@@ -192,7 +193,7 @@ message AllocatedTaskInfo {
 message TaskListInfo {
     bytes namespaceId = 1;
     string name = 2;
-    int32 taskType = 3;
+    tasklist.TaskListType taskType = 3;
     // {Normal, Sticky}
     int32 kind = 5;
```

Comment:
`kind` is the next good candidate.

Reply:
great, thank you. queueing this up for the `int`->`enum` work after this pr.